### PR TITLE
fix: metamask not working when coinbase extension is installed

### PR DESCRIPTION
**Pull request by L03TJ3**
+ some text updates for the withdraw stake switch component
+ fix price impact and added some additional confirmation for high price impact 
+ added notice for potential fail/front-run tx when using low/high slippage

### CODE REVIEW

**Comment by Bleujj1** [comment]
You will have to synchronize your wallet with bech 13 and then you debug it

**Comment by L03TJ3** [comment]
@Bleujj1 Hi, sorry, but I am unsure what to do with your comment? :)

**Comment by Bleujj1** [comment]
When you are ready let me know. I can get you an ellipal bech13 link to
synchronize your wallet and debug. All you have to do is to follow the link
till you get synchronize wallet and then you click on it

**Comment by Bleujj1** [comment]
Have you resolved your issue?

File: src/hooks/useInactiveListener.ts [hunk 1]
```diff
@@ -6,16 +6,28 @@ import { injected } from '../connectors'
  * Use for network and injected - logs user in
```
Comment (sirpy) [inline src/hooks/useInactiveListener.ts:6]: we can delete this file right?

**Review by sirpy — APPROVED** [review]

File: src/hooks/useInactiveListener.ts (hunk 1 above)
Comment (L03TJ3) [reply src/hooks/useInactiveListener.ts:6]: yes we could actually. I'll do it in one of my next commits :)

**Pull request merged**